app/views/list/edit.html.slim

```diff
@@ -0,0 +1 @@
+h1 = "Hello this is the edit page..."
```

Comment:
You should just be able to reuse the create page in the edit page - to get the header right you can just do `@list.new_record? ? "Create" : "Edit"`